MIGRATION-FROM-SCK.md

```diff
@@ -0,0 +1,325 @@
+
+#### Pod Security Context
+If you are using the podSecurityContext option to set a pod security policy, you can use the podSecurityContext option in Splunk OpenTelemetry Collector for Kubernetes.
+
+#### Custom Metadata
+If you are using the customMetadata option to set a custom metadata field in your logs in SCK, you can use the customMetadata option in Splunk OpenTelemetry Collector for Kubernetes.
+
+#### Custom Annotations
+If you are using the customMetadataAnnotations option to set custom annotation fields in your logs in SCK, you can use the "annotations" and "podAnnotations" option in Splunk OpenTelemetry Collector for Kubernetes.
+
+#### Pod Scheduling configs
+If you are using any pod scheduling operations such as nodeSelector, affinity and tolerations in SCK, you can use the nodeSelector, affinity and tolerations options in Splunk OpenTelemetry Collector for Kubernetes.
+
+#### Service Accounts
+If you are using the serviceAccount option to use your own service accounts in SCK, you can use the serviceAccount option in Splunk OpenTelemetry Collector for Kubernetes.
+
+#### Secrets
+If you are using the secret option to use your own service accounts in SCK, you can use the secret option in Splunk OpenTelemetry Collector for Kubernetes.
+
+#### Custom docker image and pull secrets
+If you are using custom docker images, tags, pull secrets and pull policy in SCK, you can achieve the same using the "image" option in Splunk OpenTelemetry Collector for Kubernetes to specify the relevant configs for using docker images.
+
+#### Limiting/Increasing Application Resources
+If you are using the resources option in SCK to limit/increase the CPU and memory usage in SCK, you can do the same using the resources option in Splunk OpenTelemetry Collector for Kubernetes.
+
+#### Extra Files from Host
+For tailing files other than container or journald logs (i.e., kube audit logs), you must configure "extraFileLogs" using this "[filelog](https://github.com/open-telemetry/opentelemetry-collector-contrib/tree/main/receiver/filelogreceiver)" receiver configuration. 
+
+[SCK values.yaml snippet]
+```
+logs:
+  kube-audit:
+    from:
+      file:
+        path: /var/log/kube-apiserver-audit.log
+    timestampExtraction:
+      format: "%Y-%m-%dT%H:%M:%SZ"
+    sourcetype: kube:apiserver-audit
+```    
+
+[SCK-OTEL values.yaml snippet]
+```
+extraFileLogs:
+ filelog/kube-audit:
+   include: [/var/log/kube-apiserver-audit.log]
+   start_at: beginning
+   include_file_path: true
+   include_file_name: false
+   resource:
+     service.name: /var/log/kube-apiserver-audit.log
+     host.name: 'EXPR(env("K8S_NODE_NAME"))'
+     com.splunk.sourcetype: kube:apiserver-audit
+```     
+
+You must use the same keyword ("kube-audit" in the above example) for continuing to read from the translated checkpoint data.
```

Comment:
"Use the `kube-audit` keyword to continue reading from the translated checkpoint data."